ext/oci8/oci8.stub.php

```diff
@@ -294,32 +294,42 @@ function oci_close($connection): ?bool {}
  */
 function ocilogoff($connection): ?bool {}
 
-/** @return resource|false */
+/**
+ * @sensitive-param $password
```

Comment:
Nit: please put the `@sensitive-param` tag below the `@return` (and `@param`) tags

Reply:
OK, the current order makes sense! Just make sure that `@sensitive-param` is below `@param`.

Reply:
Please clarify: Below all `@param`s or just below the corresponding `@param`?

Reply:
Hm, I'd say below all `@param`s. But as far as I see, we have precedence for both styles

Reply:
Okay, based on looking through a few of the other files I've done the following:

```
@param
@sensitive-param
@return
@alias
@refcount
@deprecated
```